### PR TITLE
refine qwen3_vl_moe experts forward

### PR DESCRIPTION
**Pull request by WeiweiZhang1**
## Description

Please briefly describe your main changes, the motivation.

## Type of Change

- [ ] Bug fix
- [ ] New feature
- [ ] Documentation update
- [x] Performance improvement
- [ ] Code refactoring
- [ ] Other (please specify):

## Related Issues

Fixes or relates to #

## Checklist Before Submitting

- [x] My code has been tested locally.
- [ ] Documentation has been updated as needed.
- [ ] New or updated tests are included where applicable.

The measured accuracy of the quantized model are the same as before.

### CODE REVIEW

File: auto_round/modeling/fused_moe/qwen3_vl_moe.py [hunk 1]
```diff
@@ -92,19 +92,23 @@ def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
 
         # convert router indices into OHE list
         # reshape to be (num_experts, top_k, batch_size * sequence_length)
-        expert_mask = torch.nn.functional.one_hot(router_indices, num_classes=self.num_experts).permute(2, 1, 0)
-
-        for expert_idx, expert_layer in enumerate(self.experts):
-            idx, token_idx = torch.where(expert_mask[expert_idx].squeeze(0))
-
-            if self.calibrate_all_experts:
+        with torch.no_grad():
```
Comment (yiliu30) [inline auto_round/modeling/fused_moe/qwen3_vl_moe.py:95]: Does that work for the tuning case?

Comment (WeiweiZhang1) [reply auto_round/modeling/fused_moe/qwen3_vl_moe.py:95]: this won't affect it.
